paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc

```diff
@@ -417,36 +417,88 @@ void TestImmutableOpBetweenNonQuantizedOp(const std::string tested_op) {
            SCALE * S8_MAX);
 }
 
+// a->Dropout1->b
+// b->TestedOp1(won't be quantized)->c
+//    c->Dropout2->d
+//    c->TestedOp2(will be quantized)->e
+//        e->Pool2d1(will be quantized)->f
+//        e->Pool2d2(will be quantized)->g
+void TestImmutableOpWithManyOutputs(const std::string tested_op) {
```

Comment:
```
/*
                         a 
                         | 
                        Dropout1
                         | 
                         b
                         |
      TestedOp1(won't be quantized)
                         |
                       c    c 
                      /      \       
               Dropout2       TestedOp2(will be quantized)          
                                            | 
                                            e
                                          /    \
                    Pool2d1(will be quantized)   Pool2d2(will be quantized)       
                            |                                         |
                            f                                         g
*/
```
If one immutable op has many next op and more than one of the next op is not quantizable, then do not quantize this immutable op ? Is it correct? 
But why TestedOp2, Pool2d1 and Pool2d2 should be not quantized? testedop2 has two followed op both quantizable.

Reply:
If one of the next op is not quantizable we don't want to quantize TestedOp2, because we will have to put `quantize` before TestedOp2 and `dequantize` after. 
Immutable ops usually have the same performance comparing FP32 and INT8.  That is why we should be careful adding `quantize`, `dequantize`. 

So in this situation, we skip quantization of this operator only when prev_op won't be quantize and not all next ops will be quantize
```
  if (!(IsOpDequantized(prev_op)) && !(IsOpQuantized(nearest_interp_out))) {
      return;
    }
```
In any other situation, we will quantize this op.